### PR TITLE
nix-update-source: 0.5.0 -> 0.6.3

### PR DESCRIPTION
**Pull request by timbertson**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/package-management/nix-update-source/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 python3Packages.buildPythonApplication rec {
-  version = "0.5.0";
+  version = "0.6.3";
   name = "nix-update-source-${version}";
   src = fetchFromGitHub {
     owner = "timbertson";
     repo = "nix-update-source";
-    rev = "version-0.5.0";
-    sha256 = "13icwk249frddsmn9albasikwp8asmgvp3jf9xj9adzh63wzh1i7";
+    rev = "version-0.6.3";
```
Comment (Mic92) [inline pkgs/tools/package-management/nix-update-source/default.nix]: Can you make this a variable? rev = "version-${version}";?

**Comment by timbertson** [comment]
Not easily - this field is written by a script (note the updateScript
attribute)

**Comment by Mic92** [comment]
The script is not capable of ignore this field?

**Comment by timbertson** [comment]
No, it generates the entire `fetchFromGitHub` block. Doing otherwise would require a whole lot of cleverness for no particular benefit (if it's automated, why does it matter that it's repetitious?)

**Pull request merged**